### DISCUSSION TITLE
Would you recommend to use this library for a small production app?

### DISCUSSION THREAD

**Discussion by teenjuna** [Brendonovich/prisma-client-rust#104] [General]
Hi! I am struggling with the complexity of using SQLx, and your library feels way more convenient! I haven't notice any stability disclaimer in the README, so I wanted to ask: how dangerous is it to use this library for a small production app? I am totally aware that it's not stable from a SEMVER perspective, and I don't mind making small fixes for a new release. I'm also not using any complex database features, mostly just CRUD operations (at least for now). What do think? How likely is it that this lib will hit stable release in the future?

**Comment by Brendonovich** [comment]
While I don't want to provide any guarantees yet, I'm fairly confident that you'll be alright to use PCR. We've been using it for a while at [Spacedrive](https://spacedrive.com) and it's been fine. As for a stable release, that likely won't happen until Prisma adopt the project as their own, which I don't see happening any time soon 😅

**Reply by teenjuna** [reply]
Thank you for your response! I will surely try it :)
